notes/returned-mail-received.vbs

```diff
@@ -376,51 +410,107 @@ IF CRF_checkbox = CHECKED THEN pending_verifs = pending_verifs & "Change Report
+'--CASE:NOTE Header doesn't look funky------------------------------------------05/11/2022
+'--Leave CASE:NOTE in edit mode if applicable-----------------------------------06/14/2022
+'-----General Supports-------------------------------------------------------------------------------------------------------------
+'--Check_for_MAXIS/Check_for_MMIS reviewed--------------------------------------05/11/2022
+'--MAXIS_background_check reviewed (if applicable)------------------------------05/11/2022
+'--PRIV Case handling reviewed -------------------------------------------------05/11/2022
+'--Out-of-County handling reviewed----------------------------------------------05/11/2022	discussed with Ilse
+'--script_end_procedures (w/ or w/o error messaging)----------------------------05/11/2022
+'--BULK - review output of statistics and run time/count (if applicable)--------05/11/2022
+'
+'-----Statistics--------------------------------------------------------------------------------------------------------------------
+'--Manual time study reviewed --------------------------------------------------05/14/2022
+'--Incrementors reviewed (if necessary)-----------------------------------------N/A
+'--Denomination reviewed -------------------------------------------------------N/A
+'--Script name reviewed---------------------------------------------------------06/14/2022
+'--BULK - remove 1 incrementor at end of script reviewed------------------------N/A
+
+'-----Finishing up------------------------------------------------------------------------------------------------------------------
+'--Confirm all GitHub tasks are complete----------------------------------------06/14/2022
+'--comment Code-----------------------------------------------------------------06/14/2022
+'--Update Changelog for release/update------------------------------------------06/14/2022
+'--Remove testing message boxes-------------------------------------------------06/14/2022
+'--Remove testing code/unnecessary code-----------------------------------------06/14/2022
+'--Review/update SharePoint instructions----------------------------------------06/14/2022
+'--Other SharePoint sites review (HSR Manual, etc.)-----------------------------06/14/2022
+'--COMPLETE LIST OF SCRIPTS reviewed--------------------------------------------06/14/2022
+'--Complete misc. documentation (if applicable)---------------------------------06/14/2022
+'--Update project team/issue contact (if applicable)----------------------------06/14/2022
+'--Other Note-------------------------------------------------------------------The reason we updated the residential instead of the mailing address is the inibiting errors that occur when updating the panel
+'the last iteration this was discussed at length and the determinaition was made to leave the residential address and only update mailing although that has not been the best practive at Hennepin Co. I have sent 3 Pf11 about this clarification '
```

Comment:
I think would be better to note the reasoning behind having the script update the mailing address only, instead of referencing discussions that we won't remember in the future. The mailing address update allows us to send mail to the resident in the most recently reported address, without assuming the resident has moved. Since returned mail can be an indicator that a move has occurred but is not definitive. This provides the best service to the resident and most current information as known to the agency without taking action on unconfirmed information.

Reply:
removed from closing documentation